utils/trivia.py

```diff
@@ -93,7 +96,11 @@ def end(self) -> int:
     def check(self, answer) -> int:
         # Return non-zero value to indicate failure
         # Return 0 to indicate success
-        return 0
+        if (self.qAsked.answer == answer)
```

Comment:
Syntax error: if statement needs a colon at the end

Reply:
Done!